packages/reframed/src/reframed.ts

```diff
@@ -224,17 +224,14 @@ function monkeyPatchIFrameEnvironment(
 		return;
 	}
 
-	const iframeDocumentPrototype = Object.getPrototypeOf(
-		Object.getPrototypeOf(iframeDocument)
-	);
 	const mainDocument = shadowRoot.ownerDocument;
 	const mainWindow = mainDocument.defaultView!;
 
 	let updatedIframeTitle: string | undefined = undefined;
 
 	setInternalReference(iframeDocument, "body");
 
-	Object.defineProperties(iframeDocumentPrototype, {
+	Object.defineProperties(iframeDocument, {
```

Comment:
this is the only part I don't get. Why is it a good thing that we patch the instance and not the prototype? is this required? can you please help me understand?